tests/test_modbus_rtu_block.py

```diff
@@ -104,23 +104,11 @@ def test_no_response(self, mock_client):
         self.assertFalse(len(self.last_notified[DEFAULT_TERMINAL]))
         blk.stop()
 
-    @patch("{}.sleep".format(ModbusRTU.__module__))
-    @patch('minimalmodbus.Instrument')
-    def test_execute_retry_forever(self, mock_client, mock_sleep):
-        ''' Test that retries will continue forever '''
-        blk = ModbusRTU()
-        self.configure_block(blk, {})
-        blk._backoff_strategy.retry_num = 0
```

Comment:
changes to retry mixin since this code was written make this no longer a thing:
https://github.com/niolabs/nio/commit/a24354ff879ea6ae613f23595f5b5cf3d044986c